src/app/persons/person/person.component.ts

```diff
@@ -19,14 +23,17 @@ export class PersonComponent implements OnInit {
   personId: string;
   myFormElements: FormBase<any>[] = [];
   formData = FORM_DATA_PERSON;
-
+  subscriptions: Subscription[] = [];
```

Comment:
why this?  subscriptions: Subscription[] = []; .... if it is not used?

Reply:
Resolved.